### PR TITLE
[DO NOT MERGE] Draft of 2.0 /docs changes

### PR DESCRIPTION
**Pull request by scottybollinger**
Before submitting this PR:

1. Make sure you are PR'ing from a fork, please do not push branches directly
   to this repository.
2. Ensure you've written sufficient tests for your work.
3. Double check that your changes will not break existing connectors (if
   applicable).
4. Double check that your changes do not break the `react-search-ui-views`
   storybook (if applicable).

### CODE REVIEW

**Review by chriscressman — APPROVED** [review]

**Comment by scottybollinger** [comment]
Thanks @chriscressman. I created this to test my fork but would prefer that we have a fresh PR with only content (this PR has the builder changes). I also see that we will need a Search UI maintainer to approve this, unless you have access to override the requirement.

**Pull request closed without merging**
